Test/MainWindow.xaml.cs

```diff
@@ -75,7 +75,7 @@ private void ButtonClick(object sender, RoutedEventArgs e)
 		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
 		{
 			PropertyChangedEventHandler handler = PropertyChanged;
-		    handler?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+		    handler.Invoke(this, new PropertyChangedEventArgs(propertyName));
```

Comment:
Sorry. Thought it was a typo  (new to C#).

On Thu, Sep 3, 2015, 6:47 PM Bert Verhelst <notifications@github.com> wrote:

> In Test/MainWindow.xaml.cs
> <https://github.com/bertyhell/CodeClassifier/pull/7#discussion_r38715513>:
>
> > @@ -75,7 +75,7 @@ private void ButtonClick(object sender, RoutedEventArgs e)
> >  		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
> >  		{
> >  			PropertyChangedEventHandler handler = PropertyChanged;
> > -		    handler?.Invoke(this, new PropertyChangedEventArgs(propertyName));
> > +		    handler.Invoke(this, new PropertyChangedEventArgs(propertyName));
>
> This is a conditional operator since handler could be null. Why did you
> remove this?
>
> http://www.kunal-chowdhury.com/2014/12/csharp-6-null-conditional-operators.html?m=1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bertyhell/CodeClassifier/pull/7/files#r38715513>.
>